test/source/_posts/2008-02-03-wrong-extension.yml

```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+title: This file extension is skipped
+category: test_post_reader
+---
+
+Title says it all
```

Comment:
I would suggest to add some explanation here and [submitted a corresponding PR](https://github.com/lxxxvi/jekyll/pull/1) to you.